src/effektManager.ts

```diff
@@ -336,7 +336,17 @@ export class EffektManager {
 
         const response = await vscode.window.showInformationMessage(message, 'Yes', 'No');
         if (response === 'Yes') {
-            return this.installOrUpdateEffekt(version, action);
+            const installedVersion = await this.installOrUpdateEffekt(version, action);
+            if(!!installedVersion){
+                // After installation or update is complete, offer to open the changelog
+                const changelogMessage = `Effekt ${installedVersion} has been updated. Would you like to view the changelog?`;
+                const changelogResponse = await vscode.window.showInformationMessage(changelogMessage, 'Yes', 'No');
+
+                if (changelogResponse === 'Yes') {
+                    const changelogUrl = `https://github.com/effekt-lang/effekt/releases/tag/v${installedVersion}`;
+                    vscode.env.openExternal(vscode.Uri.parse(changelogUrl));
+                }
```

Comment:
👀 
It's not always an update, sometimes it's an install (see fn header).
Not sure if we should show the changelog in that case. WDYT?

Anyways, I think that it would be nice to try and make the buttons a little bit more descriptive here -- the editor already shows a lot of "Yes" and "No" buttons that become slightly meaningless if there's 10 of them in a row :)
```suggestion
            const actionCompleted = action === 'update' ? 'updated' : 'installed';
            const changelogMessage = `Effekt ${installedVersion} has been ${actionCompleted}`;
            
            const changelogResponse = await vscode.window.showInformationMessage(changelogMessage, 'View Release Notes', 'Close');

            if (changelogResponse === 'View Release Notes') {
                const changelogUrl = `https://github.com/effekt-lang/effekt/releases/tag/v${installedVersion}`;
                vscode.env.openExternal(vscode.Uri.parse(changelogUrl));
            }
```

Reply:
I haven't tested this, but I'd be interested to know how it looks :)

Reply:
I think this is really good, especially avoiding another round of "yes" and "no" like I did 🙂  

I'm not sure if the changelog is needed after an install—maybe we could just link to the *Getting Started* section on the website instead? If I’m installing a new language for the first time, that feels more relevant.